core/libraries/plugin_api/EE_Register_Privacy_Policy.lib.php

```diff
@@ -0,0 +1,75 @@
+<?php
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
+
+
+/**
+ * Class EE_Register_Privacy_Policy
+ * Registers additional plugin privacy policy text. Yes, you could manually call `wp_add_privacy_policy_content`
+ * but you'd also need to add the logic about when to load the code etc. This way EE core takes care of knowing
+ * when to load this class and call it
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class EE_Register_Privacy_Policy implements EEI_Plugin_API
+{
+
+    /**
+     * FQCN for all privacy policy generators
+     *
+     * @var array keys are plugin_ids, and values are an array of FQCNs or FQCNs
+     */
+    protected static $privacy_policies = array();
+
+
+
+    /*
+     * @param string $plugin_id
+     * @param array $FQNSs can be the fully qualified namespaces each containing only privacy policies,
+     *              OR fully qualified class names of privacy policies
+     */
+    public static function register($plugin_id = null, $FQCNs = array())
+    {
+        self::$privacy_policies[$plugin_id] = $FQCNs;
+        // add to list of modules to be registered
+        add_filter(
+            'FHEE__EventEspresso_core_domain_services_admin_privacy_policy_Manager__privacy_policies',
```

Comment:
I think this hook name changed to:
`FHEE__EventEspresso_core_services_privacy_policy_PrivacyPolicyManager__privacy_policies`

Reply:
good catch